services/moneypenny/values-idfdev.yaml

```diff
@@ -0,0 +1,41 @@
+moneypenny:
+  host: "data-dev.lsst.cloud"
+
+  ingress:
+    enabled: true
+    hosts:
+      - host: data-dev.lsst.cloud
+        paths: ["/moneypenny"]
+    annotations:
+      nginx.ingress.kubernetes.io/auth-response-headers: X-Auth-Request-Uid, X-Auth-Request-Token
+      nginx.ingress.kubernetes.io/auth-url: "https://data-dev.lsst.cloud/auth?scope=exec:admin"
+      nginx.ingress.kubernetes.io/configuration-snippet: |
+        auth_request_set $auth_token $upstream_http_x_auth_request_token;
+        proxy_set_header Authorization "Bearer $auth_token";
```

Comment:
You should not need (or want) any of this. I'm hoping nothing needs any of this and that it's just a workaround for some old bug that we can drop.

Reply:
Removed.